Source/SwiftLintFramework/Rules/Idiomatic/NoMagicNumbersRule.swift

```diff
@@ -44,6 +44,20 @@ struct NoMagicNumbersRule: SwiftSyntaxRule, OptInRule, ConfigurationProviderRule
                 case positive = 2
                 case negative = -2
             }
+            """),
+            Example("""
+            class FooTests: XCTestCase {
+                let array: [Int] = []
+                let bar = array[42]
+            }
+            """),
+            Example("""
+            class FooTests: XCTestCase {
+                class Bar {
```

Comment:
Nested classes are discussable if they are not themselves test classes. But I agree that this rule should be disabled in test code generally. So ignoring them as well is fine.